README.md

```diff
@@ -0,0 +1,167 @@
+a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails app: Recipe-App <a name="about-project"></a>
+
+- Fully Functional Recipe App with User Authentication and Testing Using Ruby on Rails and Devise
+ Our project is a collaborative effort to develop a feature-rich recipe application built on the Ruby on Rails (ROR) framework. The primary goal of this application is to allow users to create, manage, and share their recipes while providing a seamless and intuitive user experience.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Rails
+- PostgrelSQL
+- ROR
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Key Features
+
+- User Authentication
+- Recipe Management
+- User Profile
+- Recipe Search and Filtering
+- Recipe Rating and Comments
+- User-Friendly UI
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Example command:
+
+```sh
+1.use a browser
+2.use cable internet
+3. rails routes --grep users
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+
+  git clone git@github.com:Zuwairanajma/Recipe_App.git
+  cd Recipe_App
+
+
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+
+```sh
+- Open the terminal to play with the commands.
+```
+
+### Run tests
+
+To run tests for the linter, run the following command:
+
+Example command:
+
+```sh
+rspec
+bundle exec rspec spec/system
+```
+
+```sh
+rubocop -A
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+ 👤 **[Zuwaira Sadiq]**
+
+- GitHub: [@Zuwairanajma](https://github.com/Zuwairanajma)
+- Twitter: [@SadiqJuwairiyya](https://twitter.com/SadiqJuwairiyya)
+- LinkedIn: [Zuwaira Sadiq](https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [REQUIRED] Good job so far! I apreciate what you have done but you should add all `team` members to your `README` file 🙏🏼

Reply:
- Good job done so far, just as the previous reviewer mentioned , please add the details of all team members in your readme. This would help make your repo more professional. Please fix this 👍 👍